Examples/HttpAnalyzer/main.cpp

```diff
@@ -389,21 +392,21 @@ void onApplicationInterrupted(void* cookie)
 /**
  * activate HTTP analysis from pcap file
  */
-void analyzeHttpFromPcapFile(std::string pcapFileName)
+void analyzeHttpFromPcapFile(std::string pcapFileName, unsigned short dstPort)
 {
 	// open input file (pcap or pcapng file)
 	IFileReaderDevice* reader = IFileReaderDevice::getReader(pcapFileName.c_str());
 
 	if (!reader->open())
 		EXIT_WITH_ERROR("Could not open input pcap file");
 
-	// set a port 80 filter on the reader device to process only HTTP packets
-	PortFilter httpPortFilter(80, SRC_OR_DST);
+	// set a port  filter on the reader device to process only HTTP packets
+	PortFilter httpPortFilter(dstPort, DST);

```

Comment:
the filter needs to check `SRC_OR_DST` to be able to capture both HTTP requests and responses